ansible/roles/samza-jobs/defaults/main.yml

```diff
@@ -55,6 +55,8 @@ telemetry_extractor_messages_fetch_threshold: 3000
 consumer_fetch_max_bytes: 1572864
 producer_max_request_size_bytes: 1572864
 
+dedup_include_producer_pids: sunbird-portal
```

Comment:
We should use pdata.id rather than pid. Also pdata.id is env specific.

Reply:
changed code to use pdata.id